src/main/java/net/sf/jabref/logic/specialfields/SpecialFieldLocalization.java

```diff
@@ -0,0 +1,91 @@
+package net.sf.jabref.logic.specialfields;
+
+import java.util.Objects;
+
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.model.entry.specialfields.SpecialFieldValue;
+
+public class SpecialFieldLocalization {
+
+    // this is just to satisfy our own localization tests, since they will not detect if a variable is passed into Localization.lang()
```

Comment:
I admit that this was itching me. It is now moved out of SpecialField into the gui.